guides/routing/htmlforms.html

```diff
@@ -0,0 +1,246 @@
+    <p class="sentence">We will now create an HTML page containing get and post forms</p>
+    <p class="sentence">1. Create a public directory:</p>
+    <pre><code class="language-swift">mkdir ~/HTMLForms/public</code></pre>
+    <p class="sentence">2. Move to your new directory:</p>
+    <pre><code class="language-swift">cd ~/HTMLForms/public</code></pre>
+    <p class="sentence">3. Create an HTML file:</p>
+    <pre><code class="language-swift">touch formwebpage.html</code></pre>
+    <p class="sentence">4. Open your HTML file:</p>
+    <pre><code class="language-swift">open -a Xcode.app formwebpage.html</code></pre>
+    <p class="sentence">5. Copy in the following HTML: </p>
+    <pre>&lt;!DOCTYPE html&gt;
+&lt;html&gt;
+&lt;body&gt;
+&lt;h2&gt;URL Encoded Forms&lt;/h2&gt;
+
+&lt;h4&gt;Post to Codable route&lt;/h4&gt;
+&lt;form action="codable" method="post" enctype="application/x-www-form-urlencoded" target="redirect"&gt;
+    Name: &lt;input type="text" name="name" required="required"&gt;&lt;br&gt;
+    Age: &lt;input type="number" name="age" &gt;&lt;br&gt;
+    isDeveloper:&lt;input type="checkbox" name="isDeveloper" value="true"&gt;&lt;br&gt;
+    &lt;input type="submit" value="Submit"&gt;
+&lt;/form&gt;
+
+&lt;h4&gt;Get to Codable route&lt;/h4&gt;
+&lt;form action="codable" method="get" enctype="application/x-www-form-urlencoded" target="redirect"&gt;
+    Name: &lt;input type="text" name="name" required="required"&gt;&lt;br&gt;
+    Age: &lt;input type="number" name="age" &gt;&lt;br&gt;
+    isDeveloper:&lt;input type="checkbox" name="isDeveloper" value="true"&gt;&lt;br&gt;
+    &lt;input type="submit" value="Submit"&gt;
+&lt;/form&gt;
+
+&lt;h4&gt;Post to Raw route&lt;/h4&gt;
+&lt;form action="raw" method="post" enctype="application/x-www-form-urlencoded" target="redirect"&gt;
+    Name: &lt;input type="text" name="name" required="required"&gt;&lt;br&gt;
+    Age: &lt;input type="number" name="age"&gt;&lt;br&gt;
+    isDeveloper:&lt;input type="checkbox" name="isDeveloper" value="true"&gt;&lt;br&gt;
+    &lt;input type="submit" value="Submit"&gt;
+&lt;/form&gt;
+
+&lt;h4&gt;Get to Raw route&lt;/h4&gt;
+&lt;form action="raw" method="get" enctype="application/x-www-form-urlencoded" target="redirect"&gt;
+    Name: &lt;input type="text" name="name" required="required"&gt;&lt;br&gt;
+    Age: &lt;input type="number" name="age" &gt;&lt;br&gt;
+    isDeveloper:&lt;input type="checkbox" name="isDeveloper" value="true"&gt;&lt;br&gt;
+    &lt;input type="submit" value="Submit"&gt;
+&lt;/form&gt;
+
+
+&lt;iframe name="redirect" style="display:none;"&gt;&lt;/iframe&gt;
+&lt;/body&gt;
+&lt;/html&gt;</pre>
+    <p class="sentence">This HTML will make a simple page with four HTML forms. These forms contain:</p>
+    <ul>
+      <li>"Name": a required text input</li>
+      <li>"Age": an optional number input</li>
+      <li>"isDeveloper": A checkbox, which sends "true" if checked</li>
```

Comment:
The description "A checkbox..." shouldn't start with a capital letter if the descriptions for Name and Age don't as it is inconsistent.